src/Chainweb/Pact/Backend/ChainwebPactDb.hs

```diff
@@ -758,7 +758,7 @@ handlePossibleRewind v cid bRestore hsh = do
         return $ fromIntegral txid
       where msg = "handlePossibleRewind: newChildBlock: error finding txid"
 
-    rewindBlock bh = do
+    rewindBlock bh = withSavepoint Rewind $ do
```

Comment:
How does this affect possible rollbacks within this safepoint? Is it guaranteed that those still have same effect as before?

Reply:
Actually I'm not sure now that we need this PR, because `handlePossibleRewind` is always called within `withSavepoint PreBlock`. @emmanueldenloye thoughts?

Reply:
I've given some more thought to this. I don't think we need this PR.